apps/api_web/lib/api_web/controllers/schedule_controller.ex

```diff
@@ -95,10 +95,13 @@ defmodule ApiWeb.ScheduleController do
     with {:ok, filtered} <- Params.filter_params(params, @filters, conn),
          {:ok, _includes} <- Params.validate_includes(params, @includes, conn) do
       case format_filters(filtered, conn) do
+        %{route_type: _} = filters when map_size(filters) == 2 ->
+          {:error, :only_route_type}
         filters when map_size(filters) > 1 ->
           # greater than 1 because `date` is automatically included
```

Comment:
We might want to shift this up above the `case` and reword since it now describes how multiple clauses work (the `2` in `map_size(filters) == 2` is there for the same reason as the `1` in `map_size(filters) > 1`).

Reply:
I'm not really coming up a way to shift these clauses around that results in any simpler code. At the end of the day I'll still have to handle `%{route_type: _} = filters when map_size(filters) == 2` differently than just `map_size(filters) > 1`.

Reply:
ah, ok. makes sense.